api/views.py

```diff
@@ -79,15 +82,16 @@ def get_queryset(self):
 
         Shows 1) demographics attached to responses you can view, and 2) your own demographics
         """
+        qs_ids = super().get_queryset().values_list('id', flat=True)
         studies = get_objects_for_user(self.request.user, 'studies.can_view_study_responses')
         study_ids = studies.values_list('id', flat=True)
-        return DemographicData.objects.filter(Q(response__study__id__in=study_ids) | Q(user__id=self.request.user.id)).distinct()
+        return DemographicData.objects.filter((Q(response__study__id__in=study_ids) | Q(user__id=self.request.user.id)), (Q(id__in=qs_ids))).distinct()
 
 
 class UserViewSet(FilterByUrlKwargsMixin, views.ModelViewSet):
     lookup_field = 'uuid'
     resource_name = 'users'
-    queryset = User.objects.filter(is_researcher=False).distinct()
+    queryset = User.objects.all()
```

Comment:
Alright, I might be wrong. Not a big deal. I'd just leave it the way that works.